docs/concepts/resources.md

```diff
@@ -7,20 +7,6 @@ type: "docs"
 This page contains information about various tools and technologies that are
 useful to anyone developing on Knative.
 
-## Community Resources
```

Comment:
I would maybe leave the **Community Resources** heading in because `knctl` isn't the only thing listed here (we still have `ko`).